### PR TITLE
Update to use ObjectFactory package

### PR DESCRIPTION
**Pull request by JustinMoss**

### CODE REVIEW

File: RockLib.Encryption.Tests/MultiFactory.json [hunk 1]
```diff
@@ -10,12 +10,7 @@
                                 "Name": "Default",
                                 "Algorithm": "Aes",
                                 "IVSize": "16",
-                                "Key": {
-                                    "Type": "RockLib.DataProtection.UnprotectedBase64Value, RockLib.DataProtection",
```
Comment (bfriesen) [inline RockLib.Encryption.Tests/MultiFactory.json]: I thought we got rid of the DataProtection stuff. Did we forget to update the examples?

**Comment by bfriesen** [comment]
Looks good. 👍

**Comment by derikwhittaker** [comment]
Fine by me

File: RockLib.Encryption.Tests/MultiFactory.json (hunk 1 above)
Comment (bfriesen) [reply RockLib.Encryption.Tests/MultiFactory.json]: Ah, it's the tests that didn't get updated.

**Pull request merged**